### PR TITLE
Add release notes for Metals v0.10.7

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

**Review by dos65 — APPROVED** [review]

File: website/blog/2021-09-16-tungsten.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
+---
+author: Tomasz Godzik
+title: Metals v0.10.7 - Tungsten
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.10.7, which mostly focuses on
+the support for Scala 2.12.15 and refines the indent on paste functionality.
```
Comment (ckipp01) [inline website/blog/2021-09-16-tungsten.md:9]:
```suggestion
support for Scala 2.12.15 and refines the indent on paste functionality.
```

File: website/blog/2021-09-16-tungsten.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+_TASTy is an acronym that comes from the term Typed Abstract Syntax Trees. It's
+a high-level interchange format for Scala 3, and in this document we'll refer to
+it as Tasty. Tasty files are generated by the scalac compiler, and contain all
+of the information about your source code, including the syntactic structure of
+your program, and complete information about types, positions, and even
+documentation._
+
+Thanks to @kpodsiad, Metals can now show TASTy of a currently opened file using
+the new `metals.show-tasty` command. However, keep in mind that currently, this
+feature works only for Scala 3 sources.
```
Comment (ckipp01) [inline website/blog/2021-09-16-tungsten.md:130]: Do you think we should also add that currently the clients that implement this are `nvim-metals` and the VS Code Extensios? Or do you not think it's necessary?

**Review by ckipp01 — APPROVED** [review]

File: website/blog/2021-09-16-tungsten.md (hunk 2 above)
Comment (tgodzik) [reply website/blog/2021-09-16-tungsten.md]: Added, good idea!

File: website/blog/2021-09-16-tungsten.md (hunk 1 above)
Comment (tgodzik) [reply website/blog/2021-09-16-tungsten.md]: Fixed!

**Pull request merged**